alpine/Dockerfile

```diff
@@ -0,0 +1,58 @@
+FROM httpd:2.4-alpine
+
+USER root
+# Install Prereqs
+RUN set -eux && \
+  apk --update add --no-cache --virtual .build-deps \
+    alpine-sdk \
+    autoconf \
+    automake \
+    ca-certificates \
+    file \
+    libtool \
+    libxml2-dev \
+    linux-headers \
+    pcre-dev \
+    wget && \
+  update-ca-certificates
+
+
+# Download ModSecurity
+RUN set -eux && \
+  mkdir -p /usr/share/ModSecurity && \
+  cd /usr/share/ModSecurity && \
+  wget --quiet "https://github.com/SpiderLabs/ModSecurity/releases/download/v2.9.2/modsecurity-2.9.2.tar.gz" && \
+  tar -xvzf modsecurity-2.9.2.tar.gz
+
+
+# Install ModSecurity
+RUN set -eux && \
+  cd /usr/share/ModSecurity/modsecurity-2.9.2/ && \
+  sh autogen.sh && \
+  ./configure && \
+  make && \
+  make install && \
+  make clean
+
+RUN set -eux && \
+  ln -s /usr/local/apache2/conf /etc/httpd && \
+  cd /usr/share/ModSecurity/modsecurity-2.9.2/ && \
+  mkdir -p /etc/httpd/modsecurity.d && \
+  mv modsecurity.conf-recommended  /etc/httpd/modsecurity.d/modsecurity.conf && \
+  sed -i 's#^\(\s*SecAuditLog\)\s*\S*$#\1 /proc/self/fd/2#g' /etc/httpd/modsecurity.d/modsecurity.conf && \
```

Comment:
Be aware that logging is used afterwads by ftw for automated testing.

I did the same as you, extending from alpine:httpd-2.4. It will send the logs to be available using ```docker logs```, and that is what is expected for any sane containet. But that is not what ftw expects... 

For example, when using travis for testing the CRS rules, it will fail as we need (as for now) to read logs from '/var/log/httpd/error_log'.

Reply:
Is it better to have the container follow Docker convention and put the burden on the testing to put the logs in a readable location or the opposite?

I tend to favor sticking with Docker convention because it's more likely to be adopted.

Reply:
Me too. 

I was only saying to be aware that it will broken the actual usage of this Docker in particular, for testing the CRS.

But I think its worth adapting, because my alpine docker is ~~ 130MB, vs. 900MB from owasp/modsecurity-crs, which is a huge gain.

Reply:
We can quickly take a look at modifying FTW. as for the change, Alpine should be the primary one, although it has known issues with python :-X. We can keep a separate branch for the old one.